components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/SystemEngineSession.kt

```diff
@@ -49,6 +49,7 @@ class SystemEngineSession(
     @Volatile internal var trackingProtectionPolicy: TrackingProtectionPolicy? = null
     @Volatile internal var webFontsEnabled = true
     @Volatile internal var currentUrl = ""
+    @Volatile internal var useWideViewPort: Boolean? = null
```

Comment:
Reading through that again I wonder if we should leave a short comment here why we have this flag (mentioning toggleDesktopMode)?